Tryouts/Visuals/WindowsOnly/WinDwmSourceUtils/HwndMouseInputProviderEx.cs

```diff
@@ -0,0 +1,827 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
```

Comment:
There are code snippets in here which are from Microsoft - can you please clarify which license covers those?

Reply:
Actually it is not Microsoft code - it is our own code from MSDesktop. So I assume we can use it in ComposeUI also (which is a replacement for MSDesktop).